app/slackimport.go

```diff
@@ -814,7 +829,40 @@ func (a *App) OldImportUser(team *model.Team, user *model.User) *model.User {
 	return ruser
 }
 
-func (a *App) OldImportChannel(channel *model.Channel) *model.Channel {
+func (a *App) OldImportChannel(channel *model.Channel, sChannel SlackChannel, users map[string]*model.User) *model.Channel {
+	if channel.Type == model.CHANNEL_DIRECT {
+		sc, err := a.createDirectChannel(users[sChannel.Members[0]].Id, users[sChannel.Members[1]].Id)
+		if err != nil {
+			return nil
+		}
+
+		return sc
+	}
+	if channel.Type == model.CHANNEL_GROUP {
+		members := make([]string, len(sChannel.Members))
+		if len(members) < 8 {
```

Comment:
nit: it's a little roundabout to make an array from another value and then /check/ the length of that array as a precondition. Can we just check `len(sChannel.Members)` first instead?

Reply:
Yes very clean solution, fixed.